### PR TITLE
PR #44: Shooter Without Pivot (shooterIntegration to develop)

### PR DESCRIPTION
**Pull request by Irishumanoid**
Shoots notes from fixed angle with shield.
Trello card: https://trello.com/c/sfdMtxc6/98-pr-44-shooter-without-pivot-develop-to-main

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/shooter/Shooter.java [hunk 1]
```diff
@@ -181,7 +185,7 @@ public Measure<Velocity<Distance>> calculateVelocity(double targetY, Measure<Ang
 
   public double convertToRPM(double velocity) {
     // 0.0762 meters is diameter of flywheel
-    double circumference = ShooterConstants.FlywheelDiameter * Math.PI;
+    double circumference = ShooterConstants.FlywheelDiameter * Math.PI * 60;
```
Comment (mojo626) [inline src/main/java/frc/robot/subsystems/shooter/Shooter.java:186]: What is 60 for?

Comment (Irishumanoid) [reply src/main/java/frc/robot/subsystems/shooter/Shooter.java:186]: to convert to rotations per minute, not rotations per second

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -126,7 +133,7 @@ private void updateInput() {
 
   // TODO: fill in placeholder commands with actual functionality
```
Comment (1withspaghetti) [inline src/main/java/frc/robot/RobotContainer.java:130]: Remove the TODO line if the code is now functions

File: src/main/java/frc/robot/subsystems/intake/Intake.java [hunk 3]
```diff
@@ -5,20 +5,14 @@
...
   private final DigitalInput m_linebreak;
 
   /** Creates a new ExampleSubsystem. */
   public Intake() {
-    // Roller
-    m_shooterRollerMotor =
-        new CANSparkMax(ShooterConstants.kRollerMotorLeftId, MotorType.kBrushless);
     m_intakeRollerMotor = new CANSparkMax(IntakeConstants.kMotorID, MotorType.kBrushless);
     // TODO maybe use to terminate intake command
     m_linebreak = new DigitalInput(IntakeConstants.kLineBreakSensor);
```
Comment (1withspaghetti) [inline src/main/java/frc/robot/subsystems/intake/Intake.java:18]: Should the line break sensor be in the indexer subsystem if it is mounted on that part of the robot?

**Review by 1withspaghetti — APPROVED** [review]
There were two random nitpicking things I had to comment on, but if this is tested and working I'm good with it.

File: src/main/java/frc/robot/commands/shooter/Shoot.java [hunk 4]
```diff
@@ -1,26 +1,25 @@
...
 public class Shoot extends Command {
-  private final Intake m_intake;
+  private final Indexer m_indexer;
   private boolean m_reverse;
 
-  public Shoot(Intake intake, boolean reverse) {
-    m_reverse = reverse;
-    m_intake = intake;
+  public Shoot(Indexer indexer, boolean reverse) {
+    m_indexer = indexer;
```
Comment (mojo626) [inline src/main/java/frc/robot/commands/shooter/Shoot.java:13]: m_reverse is not set to a value anymore, and reverse is not used

File: src/main/java/frc/robot/commands/shooter/Aim.java
Comment (cdvdmackey) [inline src/main/java/frc/robot/commands/shooter/Aim.java:1]: drive asked us to go full manual with shield?

**Review by mojo626 — CHANGES_REQUESTED** [review]

**Review by mojo626 — APPROVED** [review]
Everything looks good, we just need to make sure to test this code on the practice field with a note through it

File: src/main/java/frc/robot/RobotContainer.java
Comment (cdvdmackey) [inline src/main/java/frc/robot/RobotContainer.java:1]: what is the todo for?

**Review by cdvdmackey — APPROVED** [review]

**Pull request merged**